create-milestone/action.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Create Milestone Regularly'
+description: 'Manupate Github API'
+inputs:
+  start-date:
+    description: 'Start date'
+    required: true
+  period-day:
```

Comment:
Instead of duration, we should use `due-on` since that is the format that milestone requires